app/code/core/Mage/Page/etc/system.xml

```diff
@@ -108,14 +108,22 @@
                             <show_in_store>1</show_in_store>
                         </includes>
                         <demonotice translate="label comment">
-                            <label>Display Demo Store Notice</label>
+                            <label>Display Store Notice</label>
                             <frontend_type>select</frontend_type>
                             <source_model>adminhtml/system_config_source_yesno</source_model>
                             <sort_order>80</sort_order>
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </demonotice>
+                        <demonotice_text translate="label comment">
+                            <label>Store Notice Text</label>
+                            <sort_order>90</sort_order>
```

Comment:
```suggestion
                            <sort_order>90</sort_order>
                            <frontend_type>textarea</frontend_type>
```

Reply:
Whats the purpose of it?

Reply:
To expand the use cases and provide flexibility for HTML and fancy store notice with links etc. As an example use case:

```html
<span class="notification-content"><b>Notice to all Users:</b> <br> We will be performing a planned maintenance  from 09:00 PM to 09:30 PM on 23rd December 2024 GMT+8. During this time the store will not be accessible.
</span>
```